LICENSE.md

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2015, Vinta Software Studio
+Copyright (c) 2013, Charles Denton and Marc Tamlyn
```

Comment:
> Redistributions of source code must retain the above copyright notice...

I think since we used the source code from ccbv.co.uk we actually have to keep it.